docs/integrations/destinations/langchain.md

```diff
@@ -98,4 +98,5 @@ qa = RetrievalQA.from_chain_type(llm=OpenAI(temperature=0), chain_type="stuff",
 
 | Version | Date       | Pull Request                                                  | Subject                                                                                                                                              |
 |:--------| :--------- |:--------------------------------------------------------------|:-----------------------------------------------------------------------------------------------------------------------------------------------------|
-| 0.0.1   | 2023-07-26 | [#26184](https://github.com/airbytehq/airbyte/pull/26184)     | Initial release  |
\ No newline at end of file
+| 0.0.2   | 2023-07-18 | [#26184](https://github.com/airbytehq/airbyte/pull/28398)     | Release on cloud  |
```

Comment:
nit: I think the real change is forcing Cython<3.0 since the metadata file isn't part of the release

Reply:
Fair point, I will add that